sql/sql_select.cc

```diff
@@ -673,7 +673,7 @@ setup_without_group(THD *thd, Ref_ptr_array ref_pointer_array,
 }
 
 static int
-setup_for_system_time(THD *thd, TABLE_LIST *tables, COND **where_expr, SELECT_LEX *select_lex)
+setup_for_system_time(THD *thd, TABLE_LIST *tables, COND **where_expr, SELECT_LEX *slex)
```

Comment:
It's func arg, so it doesn't matter. I shortened it because of too long expressions.